diff --git a/cato_api.graphqls b/cato_api.graphqls
--- a/cato_api.graphqls
+++ b/cato_api.graphqls
@@ -15050,6 +15050,8 @@ enum InternetFirewallActionEnum {
   PROMPT
   """ Apply Remote Browser Isolation (RBI) to the network traffic """
   RBI
+  """ Internal action returned for SUB_POLICY_SCOPE rules. """
+  SUB_POLICY
 }
 
 enum IpProtocol {
@@ -16486,6 +16488,8 @@ enum WanFirewallActionEnum {
   BLOCK
   """ Requests user confirmation to allow or block network traffic. """
   PROMPT
+  """ Internal action returned for SUB_POLICY_SCOPE rules. """
+  SUB_POLICY
 }
 
 enum WanFirewallDirectionEnum {
