--- a/cato_api.graphqls
+++ b/cato_api.graphqls
@@ -6032,6 +6032,7 @@
   publishPolicyRevision(input: PolicyPublishRevisionInput): SocketBypassPolicyMutationPayload!
   removeRule(input: SocketBypassRemoveRuleInput!): SocketBypassRuleMutationPayload!
   removeSection(input: PolicyRemoveSectionInput!): PolicySectionMutationPayload!
+  reorderPolicy(input: PolicyReorderInput!): SocketBypassPolicyMutationPayload! @beta
   updatePolicy(input: SocketBypassPolicyUpdateInput!): SocketBypassPolicyMutationPayload!
   updateRule(input: SocketBypassUpdateRuleInput!): SocketBypassRuleMutationPayload!
   updateSection(input: PolicyUpdateSectionInput!): PolicySectionMutationPayload!
@@ -6325,6 +6326,7 @@
   removeRule(input: SocketLanRemoveRuleInput!): SocketLanRuleMutationPayload!
   removeSection(input: PolicyRemoveSectionInput!): PolicySectionMutationPayload!
   removeSubPolicy(input: SocketLanRemoveSubPolicyInput!): SocketLanRemoveSubPolicyMutationPayload!
+  reorderPolicy(input: PolicyReorderInput!): SocketLanPolicyMutationPayload! @beta
   updatePolicy(input: SocketLanPolicyUpdateInput!): SocketLanPolicyMutationPayload!
   updateRule(input: SocketLanUpdateRuleInput!): SocketLanRuleMutationPayload!
   updateSection(input: PolicyUpdateSectionInput!): PolicySectionMutationPayload!
@@ -7626,6 +7628,7 @@
   publishPolicyRevision(input: PolicyPublishRevisionInput): WanNetworkPolicyMutationPayload!
   removeRule(input: WanNetworkRemoveRuleInput!): WanNetworkRuleMutationPayload!
   removeSection(input: PolicyRemoveSectionInput!): PolicySectionMutationPayload!
+  reorderPolicy(input: PolicyReorderInput!): WanNetworkPolicyMutationPayload! @beta
   updatePolicy(input: WanNetworkPolicyUpdateInput!): WanNetworkPolicyMutationPayload!
   updateRule(input: WanNetworkUpdateRuleInput!): WanNetworkRuleMutationPayload!
   updateSection(input: PolicyUpdateSectionInput!): PolicySectionMutationPayload!
