--- a/cato_api.graphqls
+++ b/cato_api.graphqls
@@ -3392,6 +3392,7 @@
   publishPolicyRevision(input: PolicyPublishRevisionInput): InternetFirewallPolicyMutationPayload!
   removeRule(input: InternetFirewallRemoveRuleInput!): InternetFirewallRuleMutationPayload!
   removeSection(input: PolicyRemoveSectionInput!): PolicySectionMutationPayload!
+  reorderPolicy(input: PolicyReorderInput!): InternetFirewallPolicyMutationPayload! @beta
   removeSubPolicy(input: InternetFirewallRemoveSubPolicyInput!): InternetFirewallRemoveSubPolicyMutationPayload!
   updatePolicy(input: InternetFirewallPolicyUpdateInput!): InternetFirewallPolicyMutationPayload!
   updateRule(input: InternetFirewallUpdateRuleInput!): InternetFirewallRuleMutationPayload!
@@ -7493,6 +7494,7 @@
   removeRule(input: WanFirewallRemoveRuleInput!): WanFirewallRuleMutationPayload!
   removeSection(input: PolicyRemoveSectionInput!): PolicySectionMutationPayload!
   removeSubPolicy(input: WanFirewallRemoveSubPolicyInput!): WanFirewallRemoveSubPolicyMutationPayload!
+  reorderPolicy(input: PolicyReorderInput!): WanFirewallPolicyMutationPayload! @beta
   updatePolicy(input: WanFirewallPolicyUpdateInput!): WanFirewallPolicyMutationPayload!
   updateRule(input: WanFirewallUpdateRuleInput!): WanFirewallRuleMutationPayload!
   updateSection(input: PolicyUpdateSectionInput!): PolicySectionMutationPayload!
